src/openforms/submissions/report.py

```diff
@@ -77,3 +77,13 @@ def confirmation_page_content(self) -> SafeString:
         # has been applied by ``Template.render``
         content = self.submission.render_confirmation_page()
         return content
+
+    @property
+    def appointment(self):
+        from openforms.appointments.renderer import AppointmentRenderer
```

Comment:
Can you please expose this in `appointments/service.py` and import it from there? That's how it's documented that different django apps interface with each other.